Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -1978,6 +1978,20 @@ def create_action(self, machine_id, request_body, overwrite_rate_limit_retry=Fal
                                                overwrite_rate_limit_retry=overwrite_rate_limit_retry)
         return response
 
+    def upload_to_library(self, data, file):
+        cmd_url = 'libraryfiles'
+        response = self.ms_client.http_request(method='POST', url_suffix=cmd_url, data=data, files=file)
```

Comment:
The documentation is specifying "multipart/form-data" but we can't specify that as a header because in the http standard, form-data need a boundary. Eg: `Content-Type:multipart/form-data; boundary=---------------------------293582696224464`. So setting it to only "multipart/form-data" will not works due to the missing boundary.
You can see a more detailed explanation here: https://stackoverflow.com/a/12385661
So we can't override the content-type header with the `headers` parameter and removing the default `application/json` is required.